package.json

```diff
@@ -7,27 +7,38 @@
     "migrate:up": "careen -A -e -c db/migrations/careen",
     "migrate:down": "careen -R -e -c db/migrations/careen",
     "migrate:create": "careen -C -c db/migrations/careen",
+    "pretest": "dropdb test-api-boilerplate || true && createdb test-api-boilerplate",
     "seed": "careen -c db/seeds/careen",
     "seed:up": "careen -A -e -c db/seeds/careen",
     "seed:down": "careen -R -e -c db/seeds/careen",
-    "seed:create": "careen -C -c db/seeds/careen"
+    "seed:create": "careen -C -c db/seeds/careen",
+    "test": "PGNAME=test-api-boilerplate mocha --compilers js:babel-register"
```

Comment:
PGNAME is a slightly awkward ENV to use to indicate TEST, BUT I'm taking cues from 12-factor not to make ENVs into fake "environment" combinations. So...I'm open to feedback on how to handle this!